src/package.json

```diff
@@ -149,7 +149,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "nyc mocha --timeout 30000 --recursive ../tests/backend/specs ../node_modules/ep_*/static/tests/backend/specs",
+    "test": "nyc mocha --timeout 120000 --recursive ../tests/backend/specs ../node_modules/ep_*/static/tests/backend/specs",
```

Comment:
Why was the timeout changed?

Reply:
Tests take more than 30 seconds on my slowest test devices here.  It doesn't need to be part of this lint branch if you want but it helps me test if it's in :)